assessments/0002-notary-project.md

```diff
@@ -0,0 +1,195 @@
+   - If so, then V1 can act as a base for the site and no separation of versions need be applied.
+
+#### Content creation processes
+
+<!-- borrowed from @celestehorgan's excellent work in 0001-contour.md -->
+- The following are ideas to consider going forward with a docs repo. As a content creator:
+   - If I write documentation, who verifies that it's technically accurate? 
+   - If I'm not a native english speaker, who can I ask for help with grammar and language review? 
+   - If I'm a trained technical writer and want to rearrange, create new, or split existing topics, who do I tag in issues/on Slack for comment?
+   - As a developer user, how can I ensure that content is accurate and up to date? Are alpha and beta features flagged? What happens when content (+ functionality) gets deprecated or changed in a major way? 
+   - Who reviews documentation PRs? 
+
+### Recommendations
+
+#### Information Architecture
+
+The assumption is that V2 is the primary project moving forward, and that a Minimum Viable Product (MVP) for a website/documentation will focus on Notary V2 users.
+- Bring the nv2 cli prototype documentation into the main documentation when conosolidating the content on the website.
+- V1 Docs will need to be reworked once an information architecture (IA) is setup for the new site. The new site IA can likely be informed by the existing V1 docs.
+
+The [Service Mesh Interface website](https://smi-spec.io/) handles its specification tab very well -- linking directly to the relevant markdown folder in GitHub, allowing website users to find the specification work where it is happening, and giving new users an opportunity to participate.
+
+
+## Contributor documentation 
+
+| Criteria                                  | 1   | 2   | 3   | 4   | 5   |
+| ---                                       | --- | --- | --- | --- | --- |
+| Communication methods documented          |     |     | ✅  |     |     |
+| Beginner friendly issue backlog           |     | ✅  |     |     |     |
+| “New contributor” getting started content | ✅  |     |     |     |     |
+| Project governance documentation          | ✅  |     |     |     |     |
+
+Criteria 
+- 1 = (Is not present or requires significant work)
+- 3 = (is present, but needs work)
+- 5 = (is executed extremely well or no improvement required) |
+
+### Comments & recommendations
+
+- Most of the issues in this section are a function of not having a website yet, and the project still being in a building phase.
+  - As the website build process starts, put together a skeleton IA that can be filled in as the project grows.
+
+- There are documented community communication methods, but it could be collected and expanded during the site development process. Prometheus has a good example of a community page: https://prometheus.io/community/. 
```

Comment:
```suggestion
- Communication methods are documented but could be further centralized and expanded during the site development process, with an end result comparable to https://prometheus.io/community/. 
```

In this sentence:
1. "it could be collected" -> the "it" pronoun is incorrect, as "community communication methods" is plural.
2. "there are documented community communication methods" is a bit awkwardly phrased. 

Try the suggestion above.